src/openms/source/FORMAT/MascotGenericFile.cpp

```diff
@@ -359,7 +361,8 @@ namespace OpenMS
         bool skip_spectrum_charges(param_.getValue("skip_spectrum_charges").toBool());
         if (!skip_spectrum_charges)
         {
-          os << "CHARGE=" << charge << "\n";
+          String cs = charge < 0 ? "-" : "+";
+          os << "CHARGE=" << charge << cs << "\n";
```

Comment:
does this break any of the other consumers we have for mgf, specifically the mascot server?

Reply:
see below - does not seem to be the case.